frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -255,13 +238,16 @@ class BitBox02 extends Component<Props, State> {
         title: this.props.t('bitbox02Wizard.stepInsertSD.insertSDcardTitle'),
         text: this.props.t('bitbox02Wizard.stepInsertSD.insertSDCard'),
       } });
-      return apiPost('devices/bitbox02/' + this.props.deviceID + '/insert-sdcard').then(({ success, errorMessage }) => {
-        this.setState({ sdCardInserted: success, waitDialog: undefined });
-        if (success) {
+      return insertSDCard(this.props.deviceID).then((response) => {
+        this.setState({
+          sdCardInserted: response.success,
+          waitDialog: undefined,
+        });
+        if (response.success) {
           return true;
         }
-        if (errorMessage) {
-          alertUser(errorMessage, { asDialog: false });
+        if (response.message) {
+          alertUser(response.message, { asDialog: false });
```

Comment:
couldn't destruct the argument directly anymore as TypeScript complained that `errorMessage` is optional.